### DISCUSSION TITLE
google-re2 installation on macOS is broken, and cascades into Airflow 2.6.3

### DISCUSSION THREAD

**Discussion by tejasanilshah** [apache/airflow#32829] [Q&A]
I was trying to set up a simple barebones conda environment with Airflow 2.6.3, but it failed because google-re2 couldn't be installed. It seems like google-re2 was added to the constraints file only at 2.6.3, since 2.6.2 doesn't list google-re2 as a constraint. There are [reports](https://github.com/google/re2/issues/437) of other people running into the same problem. So simply reporting it here as well. The release notes didn't say anything about google-re2. So what am I missing?

**Comment by potiuk** [comment]
Hello @tejasanilshah - can you please report the issue with all details here: https://github.com/conda-forge/google-re2-feedstock/issues/6 - i got the advice from conda contributor. @isuruf in https://github.com/conda-forge/conda-forge.github.io/issues/1844#issuecomment-1656346019 , that this is the right place to report the issue.

**Reply by potiuk** [reply]
(also reinstallng conda might halp - worked for some of our users)
